### PR TITLE
Replace keras with ted

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- fix https://github.com/RasaHQ/roadmap/issues/699

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
@Ghostvv Couldn't we make the `max_history` 5 the default so we can keep the initial config simple?

**Comment by Ghostvv** [comment]
Do you mean inside the class?

**Comment by wochinge** [comment]
yes

**Comment by Ghostvv** [comment]
I would like to avoid that. Max history is super important parameter in TED, therefore I wanted to factor it out explicitly, so that the default `5` would be very visible and users would question it

File: examples/concertbot/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ language: en
 pipeline: supervised_embeddings
 
 policies:
-  - name: KerasPolicy
+  - name: EmbeddingPolicy
     epochs: 200
     batch_size: 50
     max_training_samples: 300
```
Comment (Ghostvv) [inline examples/concertbot/config.yml]: I don't think this parameter exists in embedding policy

File: rasa/cli/initial_project/config.yml [hunk 2]
```diff
@@ -7,5 +7,6 @@ pipeline: supervised_embeddings
 # https://rasa.com/docs/rasa/core/policies/
 policies:
   - name: MemoizationPolicy
-  - name: KerasPolicy
+  - name: EmbeddingPolicy
+    max_history: 5
```
[the remark this answers is not in the document]
Comment (wochinge) [reply rasa/cli/initial_project/config.yml]: why the epochs? Is there also no default? Let's keep it as simple as possible?

File: examples/concertbot/config.yml (hunk 1 above)
Comment (wochinge) [reply examples/concertbot/config.yml]: checked. It's part of `Policy` which in turn calls `featurize_for_training`

File: changelog/699.misc.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+The `EmbeddingPolicy <https://rasa.com/docs/rasa/core/policies/#embedding-policy>`_
+replaces the ``KerasPolicy`` in new Rasa projects generated with ``rasa init``.
```
Comment (wochinge) [inline changelog/699.misc.rst]: recommend using this

Comment (wochinge) [reply changelog/699.misc.rst]: add to migration guide

File: rasa/cli/initial_project/config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ghostvv) [reply rasa/cli/initial_project/config.yml]: yep the default is 1, let's explicitly put some epochs here

File: docs/migration-guide.rst [hunk 4]
```diff
@@ -11,6 +11,31 @@ Migration Guide
...
+  .. code-block:: yaml
+
+    policies:
+    - ... # other policies
+    - name: EmbeddingPolicy
+      max_history: 5
+      epochs: 100
+
+  The given snippet specifies default values for the parameters ``max_history`` and
+  ``epochs``. Please see the docs of the :ref:`embedding_policy` if you want to
```
Comment (Ghostvv) [inline docs/migration-guide.rst]:
```suggestion
  ``epochs``. ``max_history`` is particularly important and strongly depends on your stories. Please see the docs of the :ref:`embedding_policy` if you want to
```

**Pull request merged**